apiclient/apiclient.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2019 Thomas Weber
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apiclient
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-hclog"
+)
+
+const (
+	PODMAN_API_VERSION = "v1.0.0"
+)
+
+type APIClient struct {
```

Comment:
Since we are in `apiclient` package I think `Client` is descriptive enough, I'd suggest that the `apiclient` package could be shortened to `api` as well

Reply:
I renamed the package to "api" and the type APIClient to "API". Further, "podmanClient2" within driver struct is now just "podman".